core/io/packet_peer_udp.cpp

```diff
@@ -206,10 +208,12 @@ void PacketPeerUDP::disconnect_shared_socket() {
 	close();
 }
 
-Error PacketPeerUDP::connect_to_host(const IP_Address &p_host, int p_port) {
+Error PacketPeerUDP::connect_to_host(const IP_Address &p_host, int p_port, int p_local_port) {
 	ERR_FAIL_COND_V(udp_server, ERR_LOCKED);
 	ERR_FAIL_COND_V(!_sock.is_valid(), ERR_UNAVAILABLE);
 	ERR_FAIL_COND_V(!p_host.is_valid(), ERR_INVALID_PARAMETER);
+	ERR_FAIL_COND_V_MSG(p_port < 0 || p_port > 65535, ERR_INVALID_PARAMETER, "The server port number must be set between 0 and 65535 (inclusive).");
+	ERR_FAIL_COND_V_MSG(p_local_port < 0 || p_local_port > 65535, ERR_INVALID_PARAMETER, "The local port number must be set between 0 and 65535 (inclusive).");
```

Comment:
remove `set`:
`The local port number must be between 0 and 65535 (inclusive).`

Reply:
done